wiki-webui/src/main/java/org/exoplatform/wiki/commons/Utils.java

```diff
@@ -157,6 +157,15 @@ public static boolean canModifyPagePermission() throws Exception {
     }
     return wikiService.canModifyPagePermission(currentPage, currentUser);
   }
+  public static boolean canPublicAndRetrictPage() throws Exception {

```

Comment:
No need to use a static method IMO. Move this code directly in UIWikiPermalinkForm.